From 8af6785435dfb5220cead6e5119ad4ba660eeeb3 Mon Sep 17 00:00:00 2001
From: Tao Bao <tbao@google.com>
Date: Fri, 8 Mar 2019 16:20:07 -0800
Subject: [PATCH 46/49] minui: Fix the wrong move of the callback.

The input_cb in ev_init() should be shared among the detected input
devices.

Test: Boot into recovery. Check key inputs.
Change-Id: I0315f6a23625bacedd333fefacbf4ac6797bc1fd
---
 minui/events.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/minui/events.cpp b/minui/events.cpp
index e02aa2e0..39cb4e1c 100644
--- a/minui/events.cpp
+++ b/minui/events.cpp
@@ -102,7 +102,7 @@ int ev_init(ev_callback input_cb, bool allow_touch_inputs) {
       }
 
       ev_fdinfo[ev_count].fd = fd;
-      ev_fdinfo[ev_count].cb = std::move(input_cb);
+      ev_fdinfo[ev_count].cb = input_cb;
       ev_count++;
       ev_dev_count++;
       if (ev_dev_count == MAX_DEVICES) break;
-- 
2.17.1

